tests/spec/core/xref-spec.js

```diff
@@ -799,6 +799,37 @@ describe("Core — xref", () => {
       );
     });
 
+    it("links idl primitives that have spaces", async () => {
+      const body = `
+      <section id="test">
+        {{
+          unsigned
+          long
+          long
+        }}
+        {{  unrestricted   float
+        }}
+        {{double}}
```

Comment:
These fall back to "base" type in the inline parser... just here for regression testing.